(node:1) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.


Found an command => clear


Found an command => eval


Found an command => help


Found an command => ranklist


und an command => rntalk


Found an command => say


Found an command => usemyvoice


Found an command => voicekick


Found an command => vote


Der Websocket läuft auf dem Port: 6677


Eat and Repeat#0077 is now online


Database connection active!


Found an command => clear


Found an command => eval


Found an command => help


Found an command => ranklist


und an command => rntalk


Found an command => say


Found an command => usemyvoice


Found an command => voicekick


Found an command => vote


Der Websocket läuft auf dem Port: 6677


Database connection active!


Eat and Repeat#0077 is now online


Found an command => clear


Found an command => eval


Found an command => help


Found an command => ranklist


und an command => rntalk


Found an command => say


Found an command => usemyvoice


Found an command => voicekick


Found an command => vote


Der Websocket läuft auf dem Port: 6677


Database connection active!


Eat and Repeat#0077 is now online


(node:1) DeprecationWarning: Guild#createChannel: Create channels with an options object instead of separate parameters


(node:1) DeprecationWarning: Guild#createChannel: Create channels with an options object instead of separate parameters


(node:1) DeprecationWarning: Guild#createChannel: Create channels with an options object instead of separate parameters


(node:1) DeprecationWarning: Guild#createChannel: Create channels with an options object instead of separate parameters


(node:1) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'id' of null


    at i (/usr/src/app/commands/usemyvoice.js:12:75)


    at Object.execute (/usr/src/app/commands/usemyvoice.js:32:9)


    at Client.<anonymous> (/usr/src/app/index.js:82:36)


    at Client.emit (events.js:333:22)


    at MessageCreateHandler.handle (/usr/src/app/node_modules/discord.js/src/client/websocket/packets/handlers/MessageCreate.js:9:34)


    at WebSocketPacketManager.handle (/usr/src/app/node_modules/discord.js/src/client/websocket/packets/WebSocketPacketManager.js:105:65)


    at WebSocketConnection.onPacket (/usr/src/app/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:333:35)


    at WebSocketConnection.onMessage (/usr/src/app/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:296:17)


    at WebSocket.onMessage (/usr/src/app/node_modules/ws/lib/event-target.js:120:16)


    at WebSocket.emit (events.js:321:20)


    at Receiver.receiverOnMessage (/usr/src/app/node_modules/ws/lib/websocket.js:789:20)


    at Receiver.emit (events.js:321:20)


    at Receiver.dataMessage (/usr/src/app/node_modules/ws/lib/receiver.js:422:14)


    at Receiver.getData (/usr/src/app/node_modules/ws/lib/receiver.js:352:17)


    at Receiver.startLoop (/usr/src/app/node_modules/ws/lib/receiver.js:138:22)


    at Receiver._write (/usr/src/app/node_modules/ws/lib/receiver.js:74:10)


(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)


(node:1) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.


(node:1) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'name' of null


    at /usr/src/app/events/levelsystem/collectxp.js:23:54


    at runMicrotasks (<anonymous>)


    at processTicksAndRejections (internal/process/task_queues.js:97:5)


    at runNextTicks (internal/process/task_queues.js:66:3)


    at processImmediate (internal/timers.js:429:9)


(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)


(node:1) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'name' of null


    at /usr/src/app/events/levelsystem/collectxp.js:23:54


    at runMicrotasks (<anonymous>)


    at processTicksAndRejections (internal/process/task_queues.js:97:5)


    at runNextTicks (internal/process/task_queues.js:66:3)


    at processImmediate (internal/timers.js:429:9)


(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)


(node:1) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'name' of null


    at /usr/src/app/events/levelsystem/collectxp.js:23:54


    at runMicrotasks (<anonymous>)


    at processTicksAndRejections (internal/process/task_queues.js:97:5)


    at runNextTicks (internal/process/task_queues.js:66:3)


    at processImmediate (internal/timers.js:429:9)


(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)


(node:1) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'name' of null


    at /usr/src/app/events/levelsystem/collectxp.js:23:54


    at runMicrotasks (<anonymous>)


    at processTicksAndRejections (internal/process/task_queues.js:97:5)


    at runNextTicks (internal/process/task_queues.js:66:3)


    at processImmediate (internal/timers.js:429:9)


(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)


(node:1) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'name' of null


    at /usr/src/app/events/levelsystem/collectxp.js:23:54


    at runMicrotasks (<anonymous>)


    at processTicksAndRejections (internal/process/task_queues.js:97:5)


    at runNextTicks (internal/process/task_queues.js:66:3)


    at processImmediate (internal/timers.js:429:9)


(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 6)


(node:1) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'name' of null


    at /usr/src/app/events/levelsystem/collectxp.js:23:54


    at runMicrotasks (<anonymous>)


    at processTicksAndRejections (internal/process/task_queues.js:97:5)


(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 7)


      
      
    
